src/components/HTMLEngineProvider/BaseHTMLEngineProvider.tsx

```diff
@@ -80,6 +80,11 @@ function BaseHTMLEngineProvider({textSelectable = false, children, enableExperim
             'mention-user': HTMLElementModel.fromCustomModel({tagName: 'mention-user', contentModel: HTMLContentModel.textual}),
             'mention-report': HTMLElementModel.fromCustomModel({tagName: 'mention-report', contentModel: HTMLContentModel.textual}),
             'mention-here': HTMLElementModel.fromCustomModel({tagName: 'mention-here', contentModel: HTMLContentModel.textual}),
+            'task-title': HTMLElementModel.fromCustomModel({
```

Comment:
Do you see a need for changing this to a `markdown-title` or a generic renderer so that we can reuse the logic, instead of tagging it to task title?